PixelHunter1995/InventoryLib/Inventory.cs

```diff
@@ -150,5 +154,23 @@ public int ZIndex()
         {
             return 0; // Not drawn by scene, value here does not matter
         }
+
+        public bool GetDogAtCursor(InputManager input, out IDog dogAtCursor)
```

Comment:
There is an argument for keeping the name (and signature) same as it is in CursorStatus's helper, but I also feel like there is an argument for making it different. Particularly, to make it perhaps take Coord instead, turning it into a GetDogAtPos or the like.